### ISSUE TITLE
Support of React Native Tools in Visual Studio Code

### ISSUE THREAD

**Issue by Steviey** [start-react/native-starter-kit#61]
Support of React Native Tools in Visual Studio Code

Got red screen with "Unable to connect with remote debugger".

**Comment by SupriyaKalghatgi** [comment]
@Steviey Have you tried doing this with your earlier projects?

**Comment by Steviey** [comment]
@SupriyaKalghatgi 
Yes, it works with some other projects (not your seed). For example with a poor, naked react native project.

**Comment by SupriyaKalghatgi** [comment]
@Steviey Can you please list out the tool
